scripts/merge_tables.R

```diff
@@ -7,8 +7,12 @@
 #################
 
 # Import required packages
-if ( suppressWarnings(suppressPackageStartupMessages(require("optparse"))) == FALSE ) { stop("[ERROR] Package 'optparse' required! Aborted.") }
-if ( suppressWarnings(suppressPackageStartupMessages(require("dplyr"))) == FALSE ) { stop("[ERROR] Package 'dplyr' required! Aborted.") }
+if (suppressWarnings(suppressPackageStartupMessages(require("optparse"))) == FALSE) {
+  stop("[ERROR] Package 'optparse' required! Aborted.")
+}
+if (suppressWarnings(suppressPackageStartupMessages(require("dplyr"))) == FALSE) {
+  stop("[ERROR] Package 'dplyr' required! Aborted.")
+}
```

Comment:
Refactoring/linting was not really the point of this issue. It's best not to mix these up, as it will make it harder to focus on the changes to fix an issue, as well as for reviewing. But okay in this case, as it's not that much...